docker/celery.docker

```diff
@@ -0,0 +1,14 @@
+FROM python:3.11-slim
+
+ARG REQFILE=requirements.txt
+
+ENV PYTHONUNBUFFERED 1
+ENV PIP_NO_CACHE_DIR=1
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+
+RUN mkdir /.fs /app
```

Comment:
I'm not sure this `mkdir` is needed if you also define the volume in docker-compose. BTW, did you test that data in this volume written by the app is accessible in the celery container?